purrplingcat/PurrplingMod
# Added harmony and fixed sword swing animation

## Types of changes

- [ ] Content updates (new dialogues added or some string edited and etc)
- [ ] **Merge ASAP**
- [ ] Merge before *enumerated issues or other PRs numbers*
- [ ] Merge after *enumerated issues or other PRs numbers*

## Description
This PR adds a harmony patcher into project and fixing sword swing animation draw in right layer.

## How to test
Just recruit NPC and go to mines or secret woods. When NPC fight, check sword swing animations draws in Character layer, not above all layers.

## Checklist

**Please check if the PR fulfills these requirements**
- [ ] Changes was tested and passed

## Other information
Can be broken on Android, Linux/Mac. Needs to test.
---

Required for #20
